package.json

```diff
@@ -49,42 +51,42 @@
     ]
   },
   "devDependencies": {
-    "@types/chai": "3.4.30",
-    "@types/enzyme": "2.4.30",
-    "@types/mocha": "2.2.29",
-    "@types/react-addons-test-utils": "0.14.14",
-    "@types/sinon": "1.16.26",
-    "awesome-typescript-loader": "2.2.3",
+    "@types/chai": "3.4.34",
```

Comment:
TL;DR it makes to add `@types` dependencies the same way as you declare npm `dependencies`